quinn-proto/src/token.rs

```diff
@@ -40,12 +81,12 @@ pub(crate) struct Token {
     /// Randomly generated value, which must be unique, and is visible to the client
     pub(crate) rand: u128,
     /// Content which is encrypted from the client
-    pub(crate) inner: RetryTokenInner,
+    pub(crate) inner: TokenInner,
```

Comment:
This was already renamed once before, can we get the name right in the earlier commit?